prediction_market_agent_tooling/markets/omen/omen.py

```diff
@@ -689,6 +716,110 @@ def omen_fund_market_tx(
     market_contract.addFunding(funds_wei, from_address, from_private_key)
 
 
+def build_parent_collection_id() -> HexStr:
+    return HASH_ZERO  # Taken from Olas
+
+
+def omen_redeem_full_position_tx(
+    market: OmenAgentMarket,
+    from_address: ChecksumAddress,
+    from_private_key: PrivateKey,
+    web3: Web3 | None = None,
+    unwrap_collateral_after_redeeming_positions: bool = True,
```

Comment:
```suggestion
    unwrap_collateral_after_redeeming_positions: bool,
```

I think we shouldn't default to True in arguments that cost extra money

Reply:
nevermind, just saw the next PR, even nicer to just remove it 👍